setDeviceTokenの第二引数clietKeyに""をセットすると、アプリが異常終了します。

## 不具合の概要(Description of bug)
（お客様からの不具合報告）
[Androidのみ]setDeviceTokenの第二引数clietKeyに""をセットすると、アプリが異常終了します。 

エラーログ
``` 
05-16 14:39:27.512 3476-3814/xxxxxxxxxxxxxx E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #2 
Process: xxxxxxxxxxxxxx, PID: 3476 
java.lang.RuntimeException: An error occured while executing doInBackground() 
at android.os.AsyncTask$3.done(AsyncTask.java:309) 
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355) 
at java.util.concurrent.FutureTask.setException(FutureTask.java:222) 
at java.util.concurrent.FutureTask.run(FutureTask.java:242) 
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231) 
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
at java.lang.Thread.run(Thread.java:818) 
Caused by: java.lang.IllegalArgumentException: signature 
at com.nifty.cloud.mb.core.NCMBRequest.createSignature(NCMBRequest.java:377) 
at com.nifty.cloud.mb.core.NCMBRequest.<init>(NCMBRequest.java:338) 
at com.nifty.cloud.mb.core.NCMBRequest.<init>(NCMBRequest.java:232) 
at com.nifty.cloud.mb.core.NCMBService.sendRequestAsync(NCMBService.java:237) 
at com.nifty.cloud.mb.core.NCMBService.sendRequestAsync(NCMBService.java:318) 
at com.nifty.cloud.mb.core.NCMBInstallationService.updateInstallationInBackground(NCMBInstallationService.java:227) 
at com.nifty.cloud.mb.core.NCMBInstallation.saveInBackground(NCMBInstallation.java:652) 
at plugin.push.nifty.NiftyPushPlugin$2.done(NiftyPushPlugin.java:263) 
at com.nifty.cloud.mb.core.NCMBInstallation$1.doInBackground(NCMBInstallation.java:442) 
at com.nifty.cloud.mb.core.NCMBInstallation$1.doInBackground(NCMBInstallation.java:437) 
at android.os.AsyncTask$2.call(AsyncTask.java:297) 
at java.util.concurrent.FutureTask.run(FutureTask.java:237) 
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)? 
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)? 
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)? 
at java.lang.Thread.run(Thread.java:818)?
```
- SDKのバージョン(SDK version)
monaca_push_plugin-2.0.5
- 不具合が発生した環境(Environment requirement)
Android5
- 不具合を再現させる手順・サンプルコード(Step for Reproduce bug / Sample code)
- エラーメッセージ(Error messages)

--- a/src/android/NiftyPushPlugin.java
+++ b/src/android/NiftyPushPlugin.java
@@ -240,6 +240,10 @@ public class NiftyPushPlugin extends CordovaPlugin
         final String appKey = args.optString(0);
         final String clientKey = args.optString(1);
         final String senderId = args.optString(2);
+        if ("".equals(appKey) || "".equals(clientKey) || "".equals(senderId)) {
+            callbackContext.error("Parameters are invalid");
+            return true;
+        }
         SharedPreferences prefs = getSharedPrefs();
         SharedPreferences.Editor editor = prefs.edit();
         editor.putString(APP_KEY, appKey);

